cmake/fake_tribits.cmake

```diff
@@ -121,7 +121,7 @@ FUNCTION(KOKKOS_ADD_TEST)
     CMAKE_PARSE_ARGUMENTS(TEST
       "WILL_FAIL"
       "FAIL_REGULAR_EXPRESSION;PASS_REGULAR_EXPRESSION;EXE;NAME;TOOL"
-      "CATEGORIES;ARGS"
+      "CATEGORIES;ARGS;COMMON_ARGS"
```

Comment:
It doesn't look like ARGS is being used with COMMON_ARGS anywhere in the tests? Am I missing a usage somewhere. My instinct would be to not have this argument since there are other ways of adding a common argument to a bunch of tests, but I don't have a strong opinion.

Reply:
No they aren't used in conjunction anywhere in the tests but I needed a way to pass more than one argument to one test. `ARGS` assumes that two arguments means two separate tests w/ the same executable w/ each one of the arguments.